src/components/DropdownHeader.js

```diff
@@ -6,7 +6,7 @@ import { QUERY_TYPES } from '../queries';
 import { DropdownSelect } from './DropdownSelect';
 import { Wrapper } from './Wrapper';
 
-export const DropDownHeader = ({ query, queryVariables, data, updateListData }) => {
+export const DropdownHeader = ({ query, queryVariables, data, updateListData }) => {
```

Comment:
did you use VSCode functionalities for renaming across files, or did you do it manually?

Reply:
I used VSCode for the const, and renaming of the file.

But for git to register the file name change in capitalisation I removed and readded it manually.